docs/_content/doorontwikkeling/index.md

```diff
@@ -4,6 +4,19 @@ date: '15-7-2019'
 weight: 10
 ---
 
+## Verwacht
+
+De publicatie van de 1.1.0 versies voor alle voor Zaakgericht werken in gebruik zijnde API standaarden wordt nog in 2019 verwacht. Daarmee zullen een aantal belangrijke features aan deze API standaarden worden toegevoegd. Denk daarbij aan caching voor al deze API standaarden en ondersteuning voor bestanden groter dan 3 Gb in de Documenten API standaard. Indien gewenst is verdiepen in deze vernieuwingen nu al mogelijk in de testomgevingen van deze API standaarden:
```

Comment:
Verdiepen? Snap niet wat hier staat.

Reply:
Ik zie het daar is wat tussendoor geglipt. ga ik corrigeren.

Reply:
Volgens mij is deze regel al gecorrigeerd? Ik denk hem te snappen nl :-)

Als eea. al gecorrigeerd is kun je deze conversation resolven, dan is duidelijk dat je de opmerking verwerkt hebt.